components/licence-title.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export function LicenceTitle ({
+	displayName = null,
+	isTrial = false,
+	isB2cPartnershipLicence = false,
+}){
+	if(isB2cPartnershipLicence || isTrial) {
+		return (
+			<h1 className="ncf__header ncf__center">
+				{
+					(displayName && (`${displayName}`))
+				}
+				{
+					isB2cPartnershipLicence
+						? 'Welcome to the Financial Times'
+						: (isTrial && ('Start your free trial'))
+				}
+			</h1>
+		);
+	}
+	{
+		return renderB2BTitle(displayName);
+	}
+
+}
+
+function renderB2BTitle (displayName) {
+	const companyName = (displayName && displayName !== null) ? displayName : 'Your company';
```

Comment:
is the second test `displayName !== null` useless? I think displayName should never be null, empty string potentially but not null
```suggestion
	const companyName = displayName ? displayName : 'Your company';
	// OR 
	const companyName = displayName || 'Your company';
```
Not sure which is best

Reply:
Would be best to just set the default `Component.defaultProps.displayName = 'Your Company';` IMHO

Reply:
Made default of the displayName an empty string following Ludovic's comments